cMake/FindEigen3.cmake

```diff
@@ -1,65 +1,69 @@
-# - Try to find Eigen3 lib
+# * Try to find Eigen3 lib
 #
-# This module supports requiring a minimum version, e.g. you can do
-#   find_package(Eigen3 3.1.2)
-# to require version 3.1.2 or newer of Eigen3.
+# This module supports requiring a minimum version, e.g. you can do find_package(Eigen3 3.1.2) to
+# require version 3.1.2 or newer of Eigen3.
 #
 # Once done this will define
 #
-#  EIGEN3_FOUND - system has eigen lib with correct version
-#  EIGEN3_INCLUDE_DIR - the eigen include directory
-#  EIGEN3_VERSION - eigen version
+# * EIGEN3_FOUND - system has eigen lib with correct version
+# * EIGEN3_INCLUDE_DIR - the eigen include directory
+# * EIGEN3_VERSION - eigen version
 
-# Copyright (c) 2006, 2007 Montel Laurent, <montel@kde.org>
-# Copyright (c) 2008, 2009 Gael Guennebaud, <g.gael@free.fr>
-# Copyright (c) 2009 Benoit Jacob <jacob.benoit.1@gmail.com>
+# Copyright (c) 2006, 2007 Montel Laurent, <montel@kde.org> Copyright (c) 2008, 2009 Gael
```

Comment:
For the copyright notices a format off/on would be good.

Reply:
For comments, we can also use the syntax `# ~~~` to mark the beginning and end of a comment block that should not be reformatted. I'll try that here.